src/zcl_abapgit_sap_package.clas.abap

```diff
@@ -1,33 +1,61 @@
 CLASS zcl_abapgit_sap_package DEFINITION
-    PUBLIC CREATE PRIVATE
-    GLOBAL FRIENDS zcl_abapgit_factory.
+  PUBLIC
+  CREATE PRIVATE
+
+  GLOBAL FRIENDS zcl_abapgit_factory .
 
   PUBLIC SECTION.
-    METHODS:
-      constructor
-        IMPORTING iv_package TYPE devclass.
 
-    INTERFACES: zif_abapgit_sap_package.
+    INTERFACES zif_abapgit_sap_package .
 
+    METHODS constructor
+      IMPORTING
+        !iv_package TYPE devclass .
   PRIVATE SECTION.
-    DATA: mv_package TYPE devclass.
-
-    ALIASES:
-      create FOR zif_abapgit_sap_package~create,
-      create_local FOR zif_abapgit_sap_package~create_local.
 
+    DATA mv_package TYPE devclass .
+    DATA:
+      mt_devc_buffer TYPE SORTED TABLE OF tdevc
+           WITH UNIQUE KEY devclass
+           WITH NON-UNIQUE SORTED KEY parent COMPONENTS parentcl .
```

Comment:
guess variable `mt_devc_buffer` is not needed anymore?

Reply:
Yes. Another manual merge error... 😭